test/unit/video.test.js

```diff
@@ -1,133 +1,145 @@
+/* eslint-env qunit */
 import videojs from '../../src/js/video.js';
 import TestHelpers from './test-helpers.js';
-import Player from '../../src/js/player.js';
 import * as Dom from '../../src/js/utils/dom.js';
-import log from '../../src/js/utils/log.js';
 import document from 'global/document';
 
-q.module('video.js');
+QUnit.module('video.js');
 
-test('should create a video tag and have access children in old IE', function(){
-  var fixture = document.getElementById('qunit-fixture');
+QUnit.test('should create a video tag and have access children in old IE', function() {
+  const fixture = document.getElementById('qunit-fixture');
 
   fixture.innerHTML += '<video id="test_vid_id"><source type="video/mp4"></video>';
 
-  var vid = document.getElementById('test_vid_id');
+  const vid = document.getElementById('test_vid_id');
 
-  ok(vid.childNodes.length === 1);
-  ok(vid.childNodes[0].getAttribute('type') === 'video/mp4');
+  QUnit.ok(vid.childNodes.length === 1);
```

Comment:
ugh, a lot of these need to change to use `equal` and have a message. Definitely not for this PR. Just didn't realize how many of these we still had.